docs/source/modin/dataframe.rst

```diff
@@ -207,3 +207,10 @@ DataFrame
     DataFrame.to_pandas
     DataFrame.to_snowflake
     DataFrame.to_snowpark
+
```

Comment:
It might make sense to call this out here, but a single API in its own rubric seems overkill. Can we maybe fold it with the previous category? We can rename it to `Snowflake specific API extensions (IO / conversion / caching)` if you'd like.